manifests/dprint.json

```diff
@@ -25,10 +25,27 @@
   },
   "license_markdown": "[MIT](https://github.com/dprint/dprint/blob/main/LICENSE)",
   "latest": {
-    "version": "0.46.1"
+    "version": "0.46.2"
```

Comment:
The update of dprint should be in a separate PR, IMO we should wait for the GHA to auto-create and merge it, then remove this commit.